apps/emqx/src/proto/emqx_proto_v1.erl

```diff
@@ -30,3 +37,22 @@ introduced_in() ->
 -spec is_running(node()) -> boolean() | {badrpc, term()}.
 is_running(Node) ->
     rpc:call(Node, emqx, is_running, []).
+
+-spec get_stats(node()) -> emqx_stats:stats() | {badrpc, _}.
```

Comment:
Q: would `emqx_rpc:badrpc()` be more precise for the error type?

Reply:
Not in this case, `emqx_rpc:badrpc()` is the error type for `gen_rpc`, for `rpc`: it's just https://www.erlang.org/doc/man/rpc.html#call-4